src/conways.rs

```diff
@@ -0,0 +1,85 @@
+use std::{fmt::Display, usize};
+
+pub struct Conways {
+    world: Vec<Vec<u8>>,
+    width: usize,
+    height: usize
+}
+
+impl Conways {
+    pub fn new(width: usize, height: usize) -> Self {
+        Self {
+            world: vec![vec![0; width]; height],
+            width,
+            height
+        }
+    }
+
+    pub fn set_alive(&mut self, row: usize, column: usize) {
+        self.world[row][column] = 1;
+    }
```

Comment:
The others said it already but an enum is much clearer

Reply:
done